### ISSUE TITLE
Migrate old hearing files

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#5123]
### Summary

**What we're after:**
Before we can retire some transition pages, we will need to migrate them

### Related issues

#5063 

### Completion criteria

- [x] These pages are migrated to current meeting page template (hearing page version)
https://transition.fec.gov/pages/hearings/coordinationshays3hearing.shtml
https://transition.fec.gov/pages/hearings/corplabor_indexp_ec.shtml
https://transition.fec.gov/pages/hearings/shaysfea09hearing.shtml

Use this page as a reference https://www.fec.gov/updates/7-29-8-25-2009-public-hearings-website-internet-communications-improvement-initiative/ 

- [x] Second pair of eyes before publishing
- [x] Ticket for transition redirect put in.

### Future work

Testing ticket and take-down ticket
